apps/site/assets/ts/components/Alerts.tsx

```diff
@@ -147,6 +147,10 @@ const alertDescription = (alert: AlertType): ReactElement<HTMLElement> => (
 const Alert = ({ alert }: { alert: AlertType }): ReactElement<HTMLElement> => {
   const [expanded, toggleExpanded] = useState(false);
   const onClick = (): void => toggleExpanded(!expanded);
+  // remove [http:// | https:// | www] from alert URL:
```

Comment:
@alicia-mbta it looks like this commit has a lot of code that should have been in the previous commit, in addition to the space fix mentioned in the commit message. 

I would do the following:

* split this commit into two, one containing the space fix, one containing all URL-related fixes. if you haven't split a commit before, this is a good guide: https://emmanuelbernard.com/blog/2014/04/14/split-a-commit-in-two-with-git/
* take the new commit with URL fixes, and squash it into https://github.com/mbta/dotcom/pull/432/commits/9345bfca5e53d9e45c29858b554c17cf8c845417
* take the following commit, https://github.com/mbta/dotcom/pull/432/commits/66d61ed92bab58c39c994d609c8a4132fb915257, which has all the tests for URL fixes, and squash that into https://github.com/mbta/dotcom/pull/432/commits/9345bfca5e53d9e45c29858b554c17cf8c845417 as well--tests should go in the same commit as the code they exercise

You should end up with two commits, one containing everything URL-related (code and tests both), the other with just this space fix.

I remember you said you were pretty familiar with rebasing but let me know if you want to pair a bit to get help in rearranging this PR.